src/systemctl/systemctl.c

```diff
@@ -3602,7 +3602,9 @@ static int start_special(int argc, char *argv[], void *userdata) {
 
         } else if (a == ACTION_KEXEC) {
                 r = load_kexec_kernel();
-                if (r < 0)
+                if (r < 0 && arg_force >= 1)
+                        log_notice("Failed to load kexec kernel, continuing without.");
+                else if (r < 0)
```

Comment:
not that it matters, but you check `r < 0` twice here... maybe:

```c
if (r < 0) {
        if (arg_force == 0) /* propagate the failure unless -f is given */
            return r;

        log_full(arg_quiet ? LOG_DEBUG : LOG_NOTICE, "…
}
```

Reply:
I know ;) I did it this way to avoid spurious indentation. I'm sure the compiler can figure this out.